client/hwlib/tests/test_data/device-tree/compatible

```diff
@@ -0,0 +1 @@
+raspberrypi,4-model-bbrcm,bcm2711
```

Comment:
Are these missing newlines intentional?

Reply:
it was copied from the RPi 4 machine (using `scp`), so the data is the same as on the original device

Reply:
Nice